Sources/GraphQLKit/Graphiti+Router.swift

```diff
@@ -1,20 +1,37 @@
-import Vapor
 import Graphiti
 import GraphQL
+import Vapor
+
+public extension RoutesBuilder {
+    func register<RootType>(
+        graphQLSchema schema: Schema<RootType, Request>,
+        withResolver rootAPI: RootType,
+        at path: PathComponent = "graphql",
+        postBodyStreamStrategy: HTTPBodyStreamStrategy = .collect
+    ) {
+        on(.POST, path, body: postBodyStreamStrategy) { request -> EventLoopFuture<Response> in
+            try request.resolveByBody(graphQLSchema: schema, with: rootAPI)
+                .flatMap { $0.encodeResponse(status: .ok, for: request) }
+        }
+        get(path) { request -> EventLoopFuture<Response> in
+            try request.resolveByQueryParameters(graphQLSchema: schema, with: rootAPI)
+                .flatMap { $0.encodeResponse(status: .ok, for: request) }
+        }
```

Comment:
This duplication could be removed so that the new `register(graphQLSchema: , withResolver:, at:)` is called within this function. This should work and compile I think, right?